src/components/popover/popover.scss

```diff
@@ -67,11 +66,16 @@
   flex-col
   flex-nowrap
   self-center
-  overflow-hidden
+  word-break
   h-full
   w-full;
 }
 
+.close-button {
```

Comment:
@asangma these were added so that the close button doesn't mess with the rounded corners of the popover.

Reply:
👀

Reply:
We might be able to remove those margins if we added a div around Action.close-button, gave that wrapper appropriate border-radius (for RTL and LTR) and `overflow: hidden`.

The wrapper would need to be flex and probably `flex: 0 0 auto`

Too much?

![image](https://user-images.githubusercontent.com/12503298/152598908-51c6fb96-7669-4459-b321-bf67afa8016a.png)

Reply:
I'll give it a shot. If I can't get it to work can I hand it off to you?